itn-eval-metrics.Rmd

```diff
@@ -158,7 +158,23 @@ collabs %>% dplyr::count(Category) %>%
                  plot.margin = unit(c(.75,.5,.5,.5), "cm")) + 
   ggplot2::xlab("")
 
-ggplot2::ggsave(file.path("plots", "itn_collaboration_types.png"), width = 7, height = 5)
+ggplot2::ggsave(file.path("plots", "itn_collaboration_types_all.png"), width = 7, height = 5)
```

Comment:
Adding a plot that splits out the ITCR funded folks collabs from the non ITCR funded